samples/project-anfield/config.ini

```diff
@@ -19,8 +19,10 @@ override=True
 # gflag parameter is for to decide whether to import gflags.
 gflag=False
 # prefix and suffix added to Jobname. By default no prefix/suffix is added.
-job_prefix=
-job_suffix=
+imported_job_prefix=
+imported_job_suffix=
+# List of jobs to be imported. Accepts list of comma separated strings.
```

Comment:
Good way to show example

Reply:
Can we add the example in documentation?

Reply:
Sure